charters/DocumentationCharter.md

```diff
@@ -0,0 +1,49 @@
+# Documentation Working Group Charter
+
+## Mission and Goals
+
+The Documentation Working Group ("Docs Working Group") serves as a support, helper, and advisory body to Jupyter Subprojects on all aspects of documentation. The core pillars of our mission:
+
+- Improve all aspects of documentation across the Jupyter ecosystem.
+- Make high quality documentation that is clear, comprehensive, inclusive, and serves the varying needs of Jupyter's diverse community.
+- Engage with the community to help users with Jupyter products, discover their needs, and connect them with relevant information, expertise and resources.
+
+The Docs Working Group will provide a place for consistent, focused, holistic efforts to be spent on documentation across the whole ecosystem. This group exists in part to provide capacity and resources to the subprojects (some of which are already suffering from a lack of resources/capacity, and more specifically to work on documentation in particular).
+
+We want all users to have positive experiences inside the Jupyter ecosystem, especially users who are learning and coming in for the first time, and users with disabilities.
+
+## Areas of Responsibility
+
+The Docs Working Group is responsible for the following:
+
+- Setting standards for documentation where needed.
+- The Docs working group will take ownership of documentation and related resources not owned by a Subproject (including the Jupyter.org website and documentation).
```

Comment:
I'd suggest clarifying what "takes ownership" means. Does this mean operational ownership, or content-level ownership? If content-level ownership are there certain kinds of content that require consultation or approval from other parts of Jupyter before they're changed?

Reply:
Thank you!

Based on feedback from the EC during the weekly meeting, we added these two to make sure our powers are broad enough to do what makes sense without needing to return to the EC to ask for additional powers.

The Docs Working Group is cross-functional and collaborative down to its bones, so we expect to tread VERY lightly with any binding standards (and frankly I don't think standards will be needed in the foreseeable future).

For taking ownership of jupyter.org and other orphaned docs/resources (as suggested in our meeting with the EC), we would take on both operational and content-level ownership where there's not already someone to tend to them. There seemed to be pretty broad agreement during the meeting that jupyter.org docs would be a good place for "cross-project" or "project wide" information about Jupyter, ALSO including contributor and maintenance info for the project for those working at the project-wide level.

If we need to modify these, we can add some extra detail to clarify.

Reply:
And for the jupyter.org docs, the Docs Working Group would be building upon what's already there (and the people/procedures already in place) and expanding where we need to, to follow this vision of the site/docs as a common place for project-wide information and contributor guides/materials that the whole project can (re)use and reference where needed.

Reply:
Makes sense - I am all for the vision, just trying to provide some experiences over the years watching the jupyter.org docs be stewarded by various folks. I think the most important thing to remember is that jupyter.org has a *large* reach, and so it is effectively a way to massively signal boost certain parts of the project.

With that in mind, a lot of the tension has come from questions like "which sub-projects get to have a top-level navigation item?" So just keep that kind of dynamic in mind and plan-ahead for thoughtful decision-making frameworks to navigate those topics! I don't think it's anything insurmountable, but just good to front-load some decision-making and process infrastructure so the docs working group doesn't get caught in a bind unnecessarily.

Reply:
echoing @choldgraf's concern here. 

jupyter.org has been a touchy subject in the past. Because its the most broad-reaching page in Jupyter, it's precious real estate. 

Today, it's not exactly clear _which_ group gets final say on what is added/removed from this page. In previous meetings, The EC manages the jupyter.org service (based on the governance docs), but I don't know if that means the EC is the final decision-maker on its content. 

Making the Doc's WG the group that "owns" jupyter.org, as it is worded here, seems like a big move that requires further discussion. Is this really the intention? Or are we just moving the "management", i.e. ensuring it's running and maintaining its current status, the intention?

> and expanding where we need to, to follow this vision of the site/docs as a common place for project-wide information and contributor guides/materials that the whole project can (re)use and reference where needed.

This is precisely the "touchy" point. Expanding the site is often met with debate/friction because Jupyter.org is precious real-estate. 

As an example, see the discussion on this PR previously around the top Nav Bar: https://github.com/jupyter/jupyter.github.io/pull/321#issuecomment-459566689

Reply:
Also, its worth mentioning, there has been a JEP open for a many years that proposes to re-design jupyter.org: https://github.com/jupyter/enhancement-proposals/pull/49

The reason for this proposal was that jupyter.org was—after doing a bunch of research, user interviews/testing, and experimentation—a UX team (we temporarily used a Master's program team) found that jupyter.org was becoming too busy that it was losing its utility. Building this website so that it includes _all_ the necessary docs/information for the project while remaining intuitive is actually quite hard to achieve. The re-design was a solid effort at improving this.

This would be a good thing to revisit in the working group, if the Docs WG takes over jupyter.org.